docs/units/rem.md

```diff
@@ -16,29 +22,30 @@ $base--font-size ?= 16px
 @param $px
   type: unit (pixel)
 ```
-* A pixel value passed to this mixin will be converted to an em value.
+* A pixel value passed to this mixin will be converted to a rem value.
 
 ---
 
 **Example**
 ```css
 .element
-  font-size rem(32) // 32/16 = 2
+  width rem(16 32) // 16/16 = 1, 32/16 = 2
 
 /* CSS */
 .element {
-  font-size: 2em;
+  width: 1rem 2em;
```

Comment:
Can we swap this (plus the rest in this doc) out with `margin` too, as `width` doesn't take multiple values.